v3/newrelic/trace_observer_test.go

```diff
@@ -902,7 +902,7 @@ func TestTrObsOKSendBackoffNo(t *testing.T) {
 	}
 	// If the default backoff of 15 seconds is used, the second span will not
 	// be received in time.
-	if !s.DidSpansArrive(t, 2, 4*time.Second) {
+	if !s.DidSpansArrive(t, 2, 8*time.Second) {
```

Comment:
I'd love to find a better solution to this. I've increased this timeout myself in a past PR.

Reply:
Can we set a variable for the integer here?